src/orchestrator/src/lib.rs

```diff
@@ -264,6 +264,11 @@ pub struct ServiceConfig<'a> {
     ///
     /// The orchestrator backend may or may not actually implement anti-affinity functionality.
     pub anti_affinity: Option<Vec<LabelSelector>>,
+
+    /// Whether the pod should have an ephemeral volume attached.
```

Comment:
As above, these comments should be more aware of their abstraction boundaries! The `Orchestrator` deals in generic services, not pods.

```suggestion
    /// Whether scratch disk space should be allocated for the service.
```